src/Args.php

```diff
@@ -181,11 +181,7 @@ public function getOpt($option, $default = null) {
      * @return Args Returns $this for fluent calls.
      */
     public function setOpt($option, $value) {
-        if ($value === null) {
-            unset($this->opts[$option]);
-        } else {
-            $this->opts[$option] = $value;
-        }
+        $this->opts[$option] = $value;
```

Comment:
My reasoning for doing it the other way is that if we ever want to loop over opts we want them to really be gone. Is there a reason you want to make this change?